ui/src/components/SearchControls/__tests__/SearchControls.test.js

```diff
@@ -30,8 +30,10 @@ beforeEach(() => {
 });
 
 const searchApi = new InvenioSearchApi({
-  url: documentApi.searchBaseURL,
-  withCredentials: true,
+  axios: {
```

Comment:
did it change because of axios or react-searchkit? (sorry if this was mentioned somewhere before)

Reply:
@kprzerwa searchkit :)